### DISCUSSION TITLE
Missing process or function

### DISCUSSION THREAD

**Discussion by Lunana0110** [nextflow-io/nextflow#4384] [Q&A] [answered]
I created channel using fromPath() and would like to run fastqc, but it keeps showing my process errors, I'm appreciate for any suggestions!

here is my process code:

```
process fastqc {
    
    input:
    path(reads)

    output:
    path "fastqc_res"


    script:
    """
    mkdir fastqc_res
    fastqc ${reads} -o fastqc_res

    """
}

```

**Comment by bentsherman** [comment]
Please include the entire script and error message

**Reply by Lunana0110** [reply]
Thank you for your reply!

Here is script:

```
workflow {
    ch_reads = Channel.fromPath("/Users/lsun/data/test_fastq/*.fastq")
    ch_reads.view()
    fastqc(ch_reads)
}

process fastqc {
    
    input:
    path(reads)

    output:
    path "fastqc_res"


    script:
    """
    mkdir fastqc_res
    fastqc ${reads} -o fastqc_res

    """
}

```

Here is the error message:

N E X T F L O W  ~  version 23.04.4
Launching `main.nf` [tiny_franklin] DSL2 - revision: 99379ff2ac
Missing process or function fastqc()

 -- Check script 'main.nf' at line: 13 or see '.nextflow.log' file for more details

**Reply by bentsherman** [reply, marked as answer]
I ran your script with Nextflow 23.04.4, but I wasn't able to reproduce the error

**Reply by Lunana0110** [reply]
OMG, I didn't include the process...thanks
